src/app/onboarding/review/page.tsx

```diff
@@ -0,0 +1,203 @@
+'use client';
+
+import { H1Centered, H2, H4, P } from '@/styles/text';
+import { Card, Flex } from '@/styles/containers';
+import { BigBlueButton, BigLinkButton } from '@/components/Buttons';
+import { formatTruthy, parseDate } from '@/utils/helpers';
+import { useGuardedOnboarding, useOnboardingNavigation } from '@/utils/hooks';
+import Icon from '@/components/Icon';
+import Link from 'next/link';
+import { useRouter } from 'next/navigation';
+import * as Styles from '../styles';
+
+export default function Page() {
+  const onboarding = useGuardedOnboarding();
+  const { flowAt, backlinkHref } = useOnboardingNavigation();
+  const { push } = useRouter();
+
+  // triggers on clicking submit
+  const onSubmit = () => {
+    push('/onboarding-complete');
+    onboarding.flushData();
+  };
+
+  return (
+    <Card>
+      <Flex $direction="column" $gap="30px">
+        <Styles.BackLink href={backlinkHref}>
+          <Icon type="leftArrow" />
+        </Styles.BackLink>
+
+        <H1Centered>Review & Submit</H1Centered>
+
+        {/* basic information section */}
+        <Styles.SectionBox>
+          <Flex $justify="between" $align="center">
+            <H2>Basic Information</H2>
+            <Link href={flowAt(1)}>
+              <Icon type="edit" />
+            </Link>
+          </Flex>
+          <Flex>
+            <Styles.SectionField>
+              <H4>First Name</H4>
```

Comment:
On Figma (below), all the required field labels are accompanied by an asterisk on the review page. Do we want to implement this? 
<img width="408" alt="Screen Shot 2024-04-13 at 5 05 33 PM" src="https://github.com/calblueprint/immigration-justice-project/assets/64336644/bb45af3c-830a-496d-92d0-670e7ce0fe87">

Reply:
yep, thanks for catching this!